_field-types/supported-field-types/knn-vector.md

```diff
@@ -47,6 +47,28 @@ PUT test-index
 ```
 {% include copy-curl.html %}
 
+## Vector workload modes
+
+The field of Vector Search contains a significant number of tradeoffs. Some users want very low-latency search, others want low-cost search. The `mode` mapping parameter of the `knn_vector` type allows users to indicate what tradeoffs they would like to default to. The `mode` that is set will dictate how default parameters are resolved. Users can fine tune their index by overriding other parameters in the k-NN field mapping.
+
+Right now, 2 modes are supported:
+* `in_memory` (default) - the `in_memory` mode represents the current default for vector search in OpenSearch. By default, it will use the `nmslib` engine and not configure any compression_level. This mode should be preferred if low-latency is required for your application.
+* `on_disk` - the `on_disk` mode is used to provide low-cost vector search while maintaining strong recall. The `on_disk` mode by default uses `32x` compression via binary quantization and a default rescoring oversample factor of 2.0. This mode should be used if the workload requires a lower cost. `on_disk` is only supported for `float` vector types. Because `on_disk` mode requires quantization with re-scoring, the `1x` compression level cannot be used. 
```

Comment:
nit: A table would be nice and consistent for modes with its defaults and support. The headings can be, mode, engines supported (highlight the default here), compression supported (highlight the default here) and then guidance.

```suggestion
* `on_disk` - the `on_disk` mode is used to provide low-cost vector search while maintaining strong recall. The `on_disk` mode by default uses `32x` compression via binary quantization and a default rescoring oversample factor of 3.0. This mode should be used if the workload requires a lower cost. `on_disk` is only supported for `float` vector types. Because `on_disk` mode requires quantization with re-scoring, the `1x` compression level cannot be used. 
```